web/src/styles/base/background-colors.scss

```diff
@@ -204,3 +204,6 @@
 .bg-accent-semi-cool {
   background-color: $accent-semi-cool !important;
 }
+.bg-base-lightest {
```

Comment:
[Pebble] Some of our colors are already available through USWDS/NJ's CSS file we import. I believe this can be removed.